kbase-extension/scss/partials/_custom.scss

```diff
@@ -281,14 +279,6 @@ div {
       .error-icon {
         color: rgb(255 0 0);
       }
-
-      &.input_prompt {
-        color: rgb(48 63 159);
-      }
-
-      &.output_prompt {
-        color: rgb(216 67 21);
-      }
```

Comment:
??? these aren't even visible anywhere

Reply:
Might be bleed over from some ancient template used for the vanilla Jupyter notebook.